Extension/lib/prefs.js

```diff
@@ -48,7 +48,11 @@ var Prefs = (function () {
             }
             return browser;
         })(),
-        betaYandex: navigator.userAgent.indexOf("Yowser") >= 0
+        betaYandex: navigator.userAgent.indexOf("Yowser") >= 0,
```

Comment:
to have only one string "Yowser"?
I don't know, as you can see I didn't add this line, it was here before.